sources/packages/backend/libs/services/src/students-assessments/student-assessment.model.ts

```diff
@@ -10,3 +10,13 @@ export interface StudentAssessmentDetail {
   id: number;
   originalAssessmentStudyStartDate: Date;
 }
+
+/**
+ * Full time program year student contribution for calculating program year totals.
+ */
+export enum FullTimeStudentContributionType {
+  ScholarshipBursaries = "ScholarshipBursaries",
```

Comment:
`ScholarshipBursaries` is missing a "s" from the ones in the AC (ScholarshipsBursaries).

Reply:
Wow good one